usaspending_api/common/helpers/spark_helpers.py

```diff
@@ -59,8 +60,20 @@
     DisasterEmergencyFundCode,
     SubmissionAttributes,
     DABSSubmissionWindowSchedule,
+    sam_recipient,
+
 ]
 
+BROKER_PROXY_TABLES =[
+    sam_recipient,
+    subaward,
+    published_fabs,
+    detached_award_procurement,
+    submissions,
+    appropriation,
+    object_class_program_activity,
+    award_financial,
```

Comment:
May want to comment these out as they're a little big to store as temp tables potentially in memory.

Reply:
Commented out; they are out of scope of this ticket and present a broader design question for bringing in broker tables.